downstream/modules/platform/proc-set-registry-username-password.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-set-registry-username-password"]
+
+= Setting registry_username and registry_password
+
+If you intend to use the `registry_username` and `registry_password`` variables in an inventory file you are recommended to use the following method to create a Registry Service Account to set a token with an expiration in the plaintext `inventory/vars.yml` file instead of using a plaintext username and password, for reasons of security.
```

Comment:
```suggestion
If you intend to use the `registry_username` and `registry_password` variables in an inventory file you are recommended to use the following method to create a Registry Service Account to set a token with an expiration in the plaintext `inventory/vars.yml` file instead of using a plaintext username and password, for reasons of security.
```